R/weight.R

```diff
@@ -59,3 +59,103 @@ setMethod("weightVariables", "VariableCatalog", function (x) {
         return(c())
     }
 })
+
+#' Generate a weight vector
+#'
+#' This function allows you to generate a weight vector by supplying a set of
+#' categorical variables and the target distribution for each of the variable's categories.
+#' For instance if you wanted to create a weight variable which equally weighted four categories stored
+#' in `ds$var` you would call `makeWeight(ds$var ~ c(25, 25, 25, 25), name = "weight")`.
```

Comment:
Somewhere in the docs we should say something about "rake" or "rim weighting", the methodology used. @malecki can provide the right words.